samples/basic/fade_led/src/main.c

```diff
@@ -82,6 +96,11 @@ void main(void)
 			}
 		}
 
+#if defined(CONFIG_SOC_ESP32)
+		k_sleep(MSEC_PER_SEC/50);
```

Comment:
Why is this different in ESP32?

Reply:
The same explanation for the above question.